modules/gdscript/editor/script_templates/CompositorEffect/post_process_shader.gd

```diff
@@ -0,0 +1,149 @@
+@tool
+extends CompositorEffect
+class_name PostProcessShader
```

Comment:
```suggestion
# meta-description: <Please add description here>

@tool
extends CompositorEffect
```

If the `class_name` is not necessary or desirable, then it should be avoided in the template. Otherwise see:

https://github.com/godotengine/godot/blob/71699e08c9df78b7203fa4ef9cede28e995d6ace/modules/gdscript/editor/script_templates/VisualShaderNodeCustom/basic.gd#L4-L5

https://github.com/godotengine/godot/blob/71699e08c9df78b7203fa4ef9cede28e995d6ace/modules/gdscript/editor/script_templates/RichTextEffect/default.gd#L4-L5